src/electrum_aionostr/relay.py

```diff
@@ -110,17 +110,32 @@ async def close(self, taskgroup = None):
     async def _receive_messages(self):
         while True:
             try:
-                message = await asyncio.wait_for(self.ws.receive_json(), 30.0)
-
+                message = await asyncio.wait_for(self.ws.receive_str(), 30.0)
+                # do some sanity checks
+                if len(message) > 128000:
+                    self.log.debug(f"got huge message from {self.url=}: {len(message)}")
+                    continue  # not storing or handling msg > this limit
+                message = loads(message)
```

Comment:
nit: pls `import json` and do `json.loads()` instead of `loads()`. I think that's much clearer for the reader.